packages/jsonforms-zorro-wrapper/src/controls/wysiwyg.renderer.ts

```diff
@@ -88,6 +111,26 @@ export class WysiwygRenderer extends JsonFormsControl {
       this.formats = this.uischema.options.formats || null;
     }
   }
+
+  override onChange(content: ContentChange) {
+    this.jsonFormsService.updateCore(Actions.update(this.propsPath, () => content.html));
+    if (this.uischema?.options.withStringValidation) {
+      this.jsonFormsService.updateCore(Actions.update(this.propsPath + this.stringFieldPostfix, () => content.text));
+    }
+    this.triggerValidation();
```

Comment:
I did some manual tests for that. It looks like validation for wysiwyg with `withStringValidation` set on true does not work. When I exceed the maximum length, the `form.valid` still returns `true`.

![Screenshot 2024-08-07 at 15 41 14](https://github.com/user-attachments/assets/a7a1f00f-d3b7-4fed-bc64-88b0d1dec694)

Reply:
`NOTE: Editor changes will be not applied` can you provide more details?